bin/wdq

```diff
@@ -36,41 +37,89 @@ from gwdetchar import (cli, omega, const)
 parser.add_argument('-o', '--output-directory',
                     help='output directory for scan, '
                          'default: ~/public_html/wdq/{IFO}_{gpstime}')
 parser.add_argument(
     '-f', '--config-file',
     help='path to configuration file to use, default: '
-         '~detchar/etc/omega/{epoch}/{OBS}-{IFO}_R-selected.txt')
-parser.add_argument(
+         '~detchar/etc/omega/{epoch}/{OBS}-{IFO}_R-selected.{txt/ini}')
+parser.add_argument('-w', '--wpipeline', default=omega.WPIPELINE,
+                    required=omega.WPIPELINE is None,
+                    help='path to Matlab wpipeline binary or the string '
+                         '"gwdetchar-omega", default: %(default)s')
+parser.add_argument('--colormap', default='parula',
+                    help='name of colormap to use (supported in python or '
+                         'matlab omega > r3449)')
+
+margs = parser.add_argument_group('Matlab options')
+margs.add_argument('--condor', action='store_true', default=False,
+                   help='indicates this job is running under condor, '
+                        'only use when running as part of a workflow')
+margs.add_argument(
     '-c', '--cache-file',
     help='path to data cache file, if not given, data locations '
          'are found using the datafind server, must be in LAL cache format')
-parser.add_argument('-w', '--wpipeline', default=omega.WPIPELINE,
-                    required=omega.WPIPELINE is None,
-                    help='path to wpipeline binary, default: %(default)s')
-parser.add_argument('--condor', action='store_true', default=False,
-                    help='indicates this job is running under condor, '
-                         'only use when running as part of a workflow')
 
-oargs = parser.add_argument_group('Omega options')
-oargs.add_argument('--colormap', default='parula',
-                   help='name of colormap to use (only supported for '
-                        'omega > r3449)')
+pargs = parser.add_argument_group('Python options')
+pargs.add_argument('-t', '--far-threshold', type=float, default=1e-10,
+                   help='White noise false alarm rate threshold for '
+                        'processing channels; default: %(default)s Hz')
+cli.add_nproc_option(pargs)
+pargs.add_argument('-v', '--verbose', action='store_true', default=False,
+                   help='print verbose output, default: False')
 
 args = parser.parse_args()
 
-print("----------------------------------------------\n"
-      "Creating omega scan for %s..." % args.gpstime)
-
 gps = args.gpstime
 gpstime = float(gps)
 ifo = args.ifo
 obs = args.ifo[0]
 
+def execute(cmd):
+    """Wrapper to execute a command with Popen and print live output to stdout
+    """
+    p = Popen(cmd, stdout=PIPE, stderr=STDOUT)
+    while True:
+        line = p.stdout.readline()
+        print(line, end='')
+        if line == '' and p.poll != None:
+            break
+    return p.returncode
```

Comment:
can we use [`subprocess.check_call`](https://docs.python.org/3/library/subprocess.html#subprocess.check_call) for this?

Reply:
...evidently, yes, we can! I just didn't know about it, but I'm playing around with it now. It seems to be better at capturing output than the method I currently have.